### ISSUE TITLE
Link to mailinglist returns 403 "Forbidden"

### ISSUE THREAD

**Issue by mirko** [keith-packard/snek#67]
The link [https://keithp.com/mailman/listinfo/snek](https://keithp.com/mailman/listinfo/snek) on [https://sneklang.org/](https://sneklang.org/) results in a direct HTTP error code 403 ("Forbidden")

**Comment by keith-packard** [comment]
Hrm. it's working for me. But, github is probably a friendlier place for bug reports anyways. I need to spend some quality time fixing mailman on that host...

**Comment by keith-packard** [comment]
Right, there are spammers out there using mailman subscription scripts to send annoying emails to victims. I had to disable external access to my mailing list web pages as a result. I need to upgrade to mailman3, which (I hope) has this issue solved somehow.
